src/pages/Home.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { useState, useEffect } from 'react'
+import axios from 'axios'
+import '../Styles/Home.css'
+
+
+const Home = () => {
+    
+    
+    const [homeArticles, setHomeArticles] = useState([])
+
+
+    useEffect(() => {
+        axios.get('https://nc-news-fz7g.onrender.com/api/articles')
+        .then(({data}) => {
+        console.log(data, '<<< consoling the useEffect')
+        setHomeArticles(data.articles)
+      
+    })
+    .catch((error)=>{
+    console.log('Error fetching articles:', error)
+  })
+}, [])
+
+console.log(homeArticles, '<<<<home articles')
```

Comment:
Try to remember to get rid of console logs from finalised code 🗑️ (there's a few dotted about here)